### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by genievot**
Adding Drupal in the list (Based on alphabetical order).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,15 +2,21 @@
...
+- [Inlets](inlets/README.md)
 - [MariaDB](mariadb/README.md)
-- [Movies](movies/README.md)
 - [MongoDB](mongodb/README.md)
+- [Movies](movies/README.md)
 - [OpenFaaS](openfaas/README.md)
 - [PostgreSQL](postgresql/README.md)
 - [RabbitMQ](rabbitmq/README.md)
 - [Redis](redis/README.md)
-- [WordPress](wordpress/README.md)
```
Comment (pchico83) [inline README.md:13]: Don't remove wordpress, it will be enable soon

File: terminal/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# GoTTY - Share your terminal as a web application
+
+> GoTTY is a simple command line tool that turns your CLI tools into web applications.
+## [Repository](https://github.com/yudai/gotty)
```
Comment (pchico83) [inline terminal/README.md:4]: Repo is here:
https://github.com/okteto/terminal

File: flux/README.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Flux
+Flux is most useful when used as a deployment tool at the end of a Continuous Delivery pipeline. Flux will make sure that your new container images and config changes are propagated to the cluster.
+
+> Head over to default Flux [Repository](https://github.com/fluxcd/flux)
```
Comment (pchico83) [inline flux/README.md:4]: It also uses:
- https://github.com/fluxcd/helm-operator
- https://github.com/fluxcd/flux-recv

**Comment by pchico83** [comment]
Could you also add a link to your blog post in the section `How to submit a new chart`?

File: README.md (hunk 1 above)
Comment (genievot) [reply README.md:13]: you can edit it!

Comment (genievot) [reply README.md:13]: that's why I added another commit to highlight it

File: terminal/README.md (hunk 2 above)
Comment (genievot) [reply terminal/README.md:4]: thanks for pointing to the right link.

File: flux/README.md (hunk 3 above)
Comment (genievot) [reply flux/README.md:4]: I kept it short, but you can add those, Will be useful.

**Comment by genievot** [comment]
I will add the link.

File: README.md (hunk 1 above)
Comment (pchico83) [reply README.md:13]: But you can deploy it from this repo with `helm install` although it is disabled in the UI.

File: flux/README.md (hunk 3 above)
Comment (pchico83) [reply flux/README.md:4]: Can you add them as part of this PR? or remove the readme altogether?

Comment (genievot) [reply flux/README.md]: I added in a new way, let me know if I should change the way they added.

**Comment by pchico83** [comment]
@genievot there is a conflict that needs to be solved before merging

**Comment by rberrelleza** [comment]
thanks for the contribution @genievot !

**Comment by genievot** [comment]
No prob @rberrelleza

**Comment by genievot** [comment]
No probs @rberrelleza .

**Pull request merged**